layouts/shortcodes/card-code.html

```diff
@@ -1,20 +1,15 @@
-<div class="td-card card">
-  {{ $lang := "" }}
-  {{ with $.Get "lang" }}
-    {{ $lang = $.Get "lang" }}
-  {{ end }}
-  {{ $highlight := "" }}
-  {{ with $.Get "highlight" }}
-    {{ $highlight = $.Get "highlight" }}
-  {{ end }}
-  {{- with $.Get "header" -}}
+{{ $lang := default "" ($.Get "lang") -}}
+{{ $highlight := default "" ($.Get "highlight") -}}
+
+<div class="td-card card border me-4">
```

Comment:
I thought that this shortcode was being dropped?

If not, then I'd suggest adding any custom styling through `cards-pace.scss`; avoid the `border` class here, and even consider moving the `me-4` into, say, `.td-card.card`.

Reply:
> I thought that this shortcode was being dropped?

See my answer in this [commen](https://github.com/google/docsy/pull/1376#issuecomment-1429333570)

Reply:
> If not, then I'd suggest adding any custom styling through `cards-pace.scss`; avoid the `border` class here, and even consider moving the `me-4` into, say, `.td-card.card`.

I agree. I will open a new issue that addresses this issue shortly.